science/panoptes_ruby_command_line.rst

```diff
@@ -0,0 +1,27 @@
+How to query Panoptes via Ruby console
+======================================
+
+Sometimes you will want to get live information from the Panoptes backend that is not available from the APIs. For staff and those with the appropriate level of access, the following approach is possible:
```

Comment:
Looks good, can you please also add a note that this is super user access (with no constraints) and that people should take great care running heavy queries / doing data manipulations.